v2/internal/pkg/mirror/mirror.go

```diff
@@ -198,9 +201,14 @@ func (o *Mirror) copy(ctx context.Context, src, dest string, opts *CopyOptions)
 		co.ReportWriter = opts.Stdout
 	}
 
-	return retry.IfNecessary(ctx, func() error {
+	var retryOpts retry.RetryOptions
```

Comment:
```suggestion
	var retryOpts retry.Options
```

nit: `retry.RetryOptions` is deprecated on `containers/common` in favor of `retry.Options`

Holding this PR in case you wan to change it.

/hold